### ISSUE TITLE
Gallery mode: Display multiple slides

### ISSUE THREAD

**Issue by JaySunSyn** [FabricElements/skeleton-carousel#26]
This is a feature request of a thumbnail gallery.

Two carousels that can control each other:
```
<skeleton-carousel selected="{{selected}}" slides-per-view="1">
<skeleton-carousel selected="{{selected}}" slides-per-view="5">
```

Something like this:

![image](https://user-images.githubusercontent.com/5872432/32169308-483b79dc-bd70-11e7-9c77-d3a6aa77a927.png)

What do you think?

**Comment by ernysans** [comment]
We can work on this. I think it can work but only for image galleries. Also we have a request to display a navigation on the middle just like in that image.

**Comment by JaySunSyn** [comment]
Oh ok, yes, in my app I use the mixins to display the navigation in the middle atm but with a property it'll be easier :)

**Comment by ratracegrad** [comment]
Are you providing the functionality to be able to display multiple slides?

**Comment by ernysans** [comment]
@ratracegrad No, the component isn't prepared to handle multiple slides at the same time
